services/network_config_manager/src/main/java/com/futurewei/alcor/netwconfigmanager/cache/HostResourceMetadataCache.java

```diff
@@ -30,6 +32,18 @@
     // Map <HostId, List<ResoruceIDType>>
     private ICache<String, ResourceMeta> hostResourceMetas;
 
+    public Transaction getTransaction() throws CacheException {
+        return hostResourceMetas.getTransaction().start();
```

Comment:
Will this hostResourceMetas.getTransaction() ensure the transaction across multiple caches? @pkommoju

Reply:
Yes it does. All caches read from or written to until the commit call at the botom from all methods called and their children after this transaction is stared are covered.